cypress/e2e/data-import/settings/unlink-from-job-profile.cy.js

```diff
@@ -0,0 +1,102 @@
+import getRandomPostfix from '../../../support/utils/stringTools';
+import { DevTeams, TestTypes, Permissions } from '../../../support/dictionary';
+import NewJobProfile from '../../../support/fragments/data_import/job_profiles/newJobProfile';
+import JobProfiles from '../../../support/fragments/data_import/job_profiles/jobProfiles';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import NewFieldMappingProfile from '../../../support/fragments/data_import/mapping_profiles/newFieldMappingProfile';
+import FieldMappingProfileView from '../../../support/fragments/data_import/mapping_profiles/fieldMappingProfileView';
+import JobProfileView from '../../../support/fragments/data_import/job_profiles/jobProfileView';
+import JobProfileEdit from '../../../support/fragments/data_import/job_profiles/jobProfileEdit';
+import ActionProfiles from '../../../support/fragments/data_import/action_profiles/actionProfiles';
+import MatchProfiles from '../../../support/fragments/data_import/match_profiles/matchProfiles';
+import NewMatchProfile from '../../../support/fragments/data_import/match_profiles/newMatchProfile';
+import { ACCEPTED_DATA_TYPE_NAMES, FOLIO_RECORD_TYPE } from '../../../support/constants';
+import Users from '../../../support/fragments/users/users';
+import ActionProfileView from '../../../support/fragments/data_import/action_profiles/actionProfileView';
+
+describe('data-import', () => {
+  describe('Settings', () => {
+    let user;
+    const actionProfile1 = {
```

Comment:
You can create an array of objects and pit in it mapping and action profiles.

Reply:
what array structure i can use when i need 3 action profiles 1 match profile 1 mapping profile and 1 job profile?

Reply:
I think you can try something like
const collectionOfActionProfiles = [
      { actionProfile: {
        typeValue: FOLIO_RECORD_TYPE.INSTANCE,
        name: `C11116 autotest action profile1 ${getRandomPostfix()}`,
      } },
      { actionProfile: {
        typeValue: FOLIO_RECORD_TYPE.INSTANCE,
        name: `C11116 autotest action profile2 ${getRandomPostfix()}`,
      } },
      { actionProfile: {
        typeValue: FOLIO_RECORD_TYPE.INSTANCE,
        name: `C11116 autotest action profile3 ${getRandomPostfix()}`,
      } },
    ];
    const matchProfile = {
      profileName: `C11116 autotest match profile ${getRandomPostfix()}`,
    };
    const mappingProfile = {
      name: `C11116 mapping profile ${getRandomPostfix()}`,
    };
    const jobProfile = {
      profileName: `C11116 autotest job profile ${getRandomPostfix()}`,
      acceptedType: ACCEPTED_DATA_TYPE_NAMES.MARC,
    };

    cy.visit(SettingsMenu.actionProfilePath);
    collectionOfActionProfiles.forEach((profile) => {
      ActionProfiles.create(profile.actionProfile, mappingProfile.name);
      ActionProfiles.checkActionProfilePresented(profile.actionProfile.name);
    });

Reply:
thanks. fixed